portfolio/src/main/webapp/orgs.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head> 
+    <meta charset="UTF-8">
+    <title>Oswaldo's Organizations</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script> 
+  </head>
+  <body>
```

Comment:
I noticed that you have this block in all of your pages -- this is the nav bar right? You can likely move this code into a shared file called nav.html and use jquery to do this:
https://stackoverflow.com/questions/38837835/include-html-in-another-html-file

Just a suggestion. You can do this in a followup or ignore :)